packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -53,24 +63,49 @@ pub fn rest_api_router<P: GraphPool>(store: Arc<P>) -> Router {
     merged_routes
         // Make sure extensions are added at the end so they are made available to merged routers.
         .layer(Extension(store))
-        .route(
-            "/api-doc/openapi.json",
+        .nest("/api-doc", Router::new().route(
+            "/openapi.json",
             get({
                 let doc = open_api_doc;
                 move || async { Json(doc) }
-            }),
+            })).route("/models/*path", get(static_schemas)),
         )
 }
 
+#[allow(
+    clippy::unused_async,
+    reason = "This route does not need async capabilities, but axum requires it in trait bounds."
+)]
+async fn static_schemas<'a>(Path(path): Path<String>) -> impl IntoResponse {
+    let path = path.trim_start_matches('/');
+
+    match STATIC_SCHEMAS.get_file(path) {
+        None => Response::builder()
+            .status(StatusCode::NOT_FOUND)
+            .body(body::boxed(Empty::new()))
+            .expect("Could not create 404 response"),
+        Some(file) => Response::builder()
+            .status(StatusCode::OK)
+            .header(
+                header::CONTENT_TYPE,
+                HeaderValue::from_str("application/json")
+                    .expect("Could not create header value for found static file"),
```

Comment:
```suggestion
                    .expect("could not create JSON content type header value"),
```

Considering the error is only on that